jetty-io/src/main/java/org/eclipse/jetty/io/SelectorManager.java

```diff
@@ -147,6 +147,10 @@ protected void execute(Runnable task)
     @ManagedAttribute(value = "Total number of keys in all selectors", readonly = true)
     public int getTotalKeys()
     {
+        if (_selectors == null || _selectors.length == 0)
```

Comment:
No, it's not `_selectors` that is `null`, it is the `selector` variable inside the loop below.